lib/wasm/WasmMainTemplatePlugin.js

```diff
@@ -66,7 +66,7 @@ function generateImportObject(module) {
 						`[${JSON.stringify(data.exportName)}]`
 				);
 			} else {
-				const params = data.description.params.map(
+				const params = data.description.signature.params.map(
```

Comment:
good catch! sorry for having missed that

Reply:
It's a bit weird to have this kind of breaking change on a patch version...

Reply:
I agree, that was my bad